Makefile

```diff
@@ -52,31 +52,28 @@ start:
 test:
 	@make lint
 	@make build
-	@echo ""
-	@make unit
-	@echo ""
-	@make functional
-	@echo ""
-	@make localization
-	@echo ""
+	@make coverage
 
 lint:
 	$(ESLINT) . --ext .js,.jsx,.json
 	$(SASSLINT) ./src/*.scss
 	$(SASSLINT) ./src/**/*.scss
 
 unit:
-	$(TAP) ./test/unit/*.js
+	$(TAP) --coverage ./test/unit/*.js
 
 functional:
-	$(TAP) ./test/functional/*.js
+	$(TAP) --coverage ./test/functional/*.js
 
 integration:
-	$(TAP) ./test/integration/*.js
+	$(TAP) --coverage ./test/integration/*.js
 
 localization:
-	$(TAP) ./test/localization/*.js
+	$(TAP) --coverage ./test/localization/*.js
+
+coverage:
+	$(TAP) --coverage ./test/{unit,functional,localization}/*.js
```

Comment:
Coveralls calculates coverage only from the last tap command run in Travis-CI and does so automatically if the coveralls environment variable is detected. For this reason it may make more sense to restructure this a bit:

```Makefile
test:
    @make lint
    @make tap

tap:
    $(TAP) ./test/{unit,functional,integration,localization}/*.js

coverage:
    $(TAP) ./test/{unit,functional,integration,localization}/*.js --coverage --coverage-report=lcov
```

With this we can still generate coverage reports locally, but don't add extra time to each test run needlessly. You can check out our configuration in [scratch-vm](https://github.com/LLK/scratch-vm/blob/develop/package.json) for how we have this configured elsewhere.